### ISSUE TITLE
SFSafariViewController is missing the bottom safe area inset after enrollment

### ISSUE THREAD

**Issue by chrisnok** [msintuneappsdk/ms-intune-app-sdk-ios#306]
**Describe the bug:**
SFSafariViewController is missing the bottom safe area inset after enrollment. I do not know if this also happens on the emulator since I am unable to enroll a device there.

**To Reproduce**
1. Open a SFSafariViewController to any website (layout OK)
2. Register and enroll the app
3. Complete enrollment by restarting the app
4. Open a SFSafariViewController to any website (layout missing bottom safe area inset)

How I open the SFSafariViewController 
```
let safariViewController = SFSafariViewController(url: URL(string: "https://login.microsoftonline.com")!)
safariViewController.modalPresentationStyle = .overFullScreen
self.present(safariViewController, animated: true)
```

**Versions:**
Devices: iPhone 13 mini and iPhone 14 Pro
iOS: 16.1.1 and 16.3
Intune SDK: 17.2.1
MSAL: 1.2.8

**Screenshots and logs:**
Before enrollment:
![before_enroll3](https://user-images.githubusercontent.com/83590140/217826168-230ee14d-5c9e-4577-9118-f432ab789581.png)
After enrollment:
![after_enroll3](https://user-images.githubusercontent.com/83590140/217826028-9837dd82-2cdf-4cf1-8f43-2c5b3c46e7ae.png)

**Comment by gastaffo** [comment]
We have a fix for this coming out in 17.3.0

**Comment by chrisnok** [comment]
Great! Looking forward to testing it.

**Comment by gastaffo** [comment]
Updating thread, we have released 17.3.0
